src/epics/dataElements.js

```diff
@@ -12,22 +10,6 @@ import * as types from '../constants/actionTypes.js'
 import { apiFetch } from '../util/api.js'
 import { getDisplayPropertyUrl } from '../util/helpers.js'
 
-// Load all data element groups
-export const loadDataElementGroups = (action$) =>
-    action$.ofType(types.DATA_ELEMENT_GROUPS_LOAD).concatMap(() =>
-        getD2()
-            .then((d2) =>
-                d2.models.dataElementGroups.list({
-                    fields: `id,${getDisplayPropertyUrl()}`,
-                    paging: false,
-                })
-            )
-            .then((groups) =>
-                setDataElementGroups(sortBy('name', groups.toArray()))
-            )
-            .catch(errorActionCreator(types.PROGRAMS_LOAD_ERROR))
```

Comment:
Can PROGRAMS_LOAD_ERROR also be removed?

Reply:
It is still used in epics/programs.js

Reply:
It is still used in epics/programs.js